src/pdfcapi.cpp

```diff
@@ -335,6 +335,14 @@ CAPYPDF_PUBLIC CAPYPDF_EC capy_dc_cmd_cm(CapyPDF_DrawContext *ctx,
     return conv_err(c->cmd_cm(m1, m2, m3, m4, m5, m6));
 }
 
+CAPYPDF_PUBLIC CAPYPDF_EC capy_dc_cmd_d(CapyPDF_DrawContext *ctx,
+                                        double *dash_array,
+                                        size_t dash_array_length,
```

Comment:
This is not mentioned anywhere but I aimed to have only integer sizes that have the same size on all platforms in the C API. So it would be `int32_t`.

I'm not sure whether and when to use 32 vs 64 bit values for arrays. Thus far I have used 32 bits because there's nothing in PDF that would require more than 2 billion entries. But maybe there is?